src/NHSD.BuyingCatalogue.Documents.API/NHSD.BuyingCatalogue.Documents.API.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <LangVersion>8</LangVersion>
```

Comment:
@assoomro I believe the default language version is C# 8 when targeting .NET Core 3.1 so I think we could remove this line without affecting the language version.

Reply:
That's true. For some reason my VS keep winging about the version of C# on some keywords, including this in the project file silences that. 
![Annotation 2020-02-06 095301](https://user-images.githubusercontent.com/37105695/73925779-757be600-48c6-11ea-8cca-53f3c4f02ebf.png)